src/pages/address/[address]/index.page.tsx

```diff
@@ -0,0 +1,16 @@
+import { AddressDashboard } from 'components/AddressDashboard'
+import { AppWrapper } from 'components/common'
+import { useRouter } from 'next/router'
+
+export default function AddressPage() {
+  const router = useRouter()
+  const { address } = router.query
+
+  if (!address) return null
```

Comment:
404 not found

Reply:
done via getServerSideProps, i think thats the only way? lemme know